stdlib/LinearAlgebra/src/diagonal.jl

```diff
@@ -427,7 +427,7 @@ function ldiv!(D::Diagonal{T}, V::AbstractMatrix{T}) where {T}
     end
     V
 end
-
+LinearAlgebra.ldiv!(x::AbstractArray,A::Diagonal,b::AbstractArray) = (x .= A.diag .\ b)
```

Comment:
* there is no need for the specifier `LinearAlgebra`
* there should be a space after each comma in the list of arguments
* should we be a bit more specific with regards to `x` and `b`? Maybe `AbstractVecOrMat`? not sure
* while you're adding `ldiv!`, let's have `rdiv!` as well!